src/pages/index.tsx

```diff
@@ -60,7 +61,8 @@ const Index: NextPage = () => {
 					content="Ledger Wallet Connect"
 				/>
 			</Head>
-			{isMounted ? (
+			<ApplicationDisabled />
```

Comment:
@cgrellard-ledger What do you think about displaying this page based on an ENV variable?
This way we could only switch it on/off from Vercel without the need to merge and then rollback the commit